### PR TITLE
*: unify_log_lib

### PR DESCRIPTION
**Pull request by jianzhiyao**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/13459

Problem Summary:
change log libary from "github.com/sirupsen/logrus" to "github.com/pingcap/log"

### Check List 

Tests 

- No code

Side effects

- No effects
### Release note 

- Unify log library

### CODE REVIEW

**Review by jianzhiyao — COMMENTED** [review]
I have already check all changed files. But the left importing of ``github.com/sirupsen/logrus`` seems have strong relation to it.

File: cmd/ddltest/column_test.go [hunk 1]
```diff
@@ -55,7 +56,7 @@ func (s *TestDDLSuite) checkAddColumn(c *C, rowID int64, defaultVal interface{},
 				// When insert a row with 3 columns, the third column value will be the first column value.
 				newInsertCount++
 			} else {
-				log.Fatalf("[checkAddColumn fail]invalid row: %v", data)
+				log.Fatal("[checkAddColumn fail]invalid row", zap.Any(`row`, data))
```
Comment (kennytm) [inline cmd/ddltest/column_test.go:59]: I find it very strange that *every* string is introduced with `` `backticks` `` even if it does not contain `\` or `"`. The rest of the code uses plain `"double quoted"` strings so let's be consistent 

```suggestion
				log.Fatal("[checkAddColumn fail]invalid row", zap.Any("row", data))
```

File: cmd/ddltest/column_test.go [hunk 2]
```diff
@@ -55,7 +56,7 @@ func (s *TestDDLSuite) checkAddColumn(c *C, rowID int64, defaultVal interface{},
 				// When insert a row with 3 columns, the third column value will be the first column value.
 				newInsertCount++
 			} else {
-				log.Fatalf("[checkAddColumn fail]invalid row: %v", data)
+				log.Fatal(`[checkAddColumn fail]invalid row`, zap.Any(`row`, data))
```
Comment (kennytm) [inline cmd/ddltest/column_test.go:59]:
```suggestion
				log.Fatal("[checkAddColumn fail]invalid row", zap.Any("row", data))
```

i mean use `"` for all normal strings not the other way around

Comment (jianzhiyao) [reply cmd/ddltest/column_test.go:59]: Oooo, i get it finally.

Comment (jianzhiyao) [reply cmd/ddltest/column_test.go]: Oooo, i get it finally. Please check again :-). thx!

File: tests/globalkilltest/global_kill_test.go [hunk 3]
```diff
@@ -258,9 +271,9 @@ func sleepRoutine(ctx context.Context, sleepTime int, conn *sql.Conn, connID uin
 	startTS := time.Now()
 	sql := fmt.Sprintf("SELECT SLEEP(%d);", sleepTime)
 	if connID > 0 {
-		log.Infof("exec: %s [on 0x%x]", sql, connID)
+		log.Info("exec sql", zap.String("sql", sql), zap.String("on connID", "0x"+strconv.FormatUint(connID, 16)))
```
Comment (kennytm) [inline tests/globalkilltest/global_kill_test.go:274]:
```suggestion
		log.Info("exec sql", zap.String("sql", sql), zap.String("connID", "0x"+strconv.FormatUint(connID, 16)))
```

File: tests/globalkilltest/global_kill_test.go [hunk 4]
```diff
@@ -302,9 +315,13 @@ func (s *TestGlobalKillSuite) killByKillStatement(c *C, db1 *sql.DB, db2 *sql.DB
...
 	c.Assert(err, IsNil)
-	log.Infof("connID2: 0x%x", connID2)
+	log.Info("connID2", zap.String("connID2", "0x"+strconv.FormatUint(connID2, 16)))
 
-	log.Infof("exec: KILL QUERY %v(0x%x) [on 0x%x]", connID1, connID1, connID2)
+	log.Info("exec: KILL QUERY",
+		zap.Uint64("connID1 value", connID1),
+		zap.String("connID1", "0x"+strconv.FormatUint(connID1, 16)),
+		zap.String("on connID2", "0x"+strconv.FormatUint(connID2, 16)),
+	)
```
Comment (kennytm) [inline tests/globalkilltest/global_kill_test.go:324]:
```go
log.Info("exec: KILL QUERY",
    zap.String("connID1", "0x"+strconv.FormatUint(connID1, 16)),
    zap.String("connID2", "0x"+strconv.FormatUint(connID2, 16)),
)
```

**Review by kennytm — COMMENTED** [review]
Rest LGTM

File: tests/globalkilltest/global_kill_test.go (hunk 4 above)
Comment (jianzhiyao) [reply tests/globalkilltest/global_kill_test.go]: Suggestions above is accepted and code is changed in the last commition. But 
idc-jenkins-ci-tidb/check_dev_2 is failed which seems not the code problem. Could you reload the testing task?
```
[2021-03-11T00:46:20.427Z] --- FAIL: TestInfo (0.33s)
[2021-03-11T00:46:20.427Z]     domain_test.go:210: etcdserver: requested lease not found
```

Comment (jianzhiyao) [reply tests/globalkilltest/global_kill_test.go]: All tests checked, and code is changed :-)

**Comment by jianzhiyao** [comment]
PTAL  @tiancaiamao

**Review by zimulala — APPROVED** [review]

**Comment by jianzhiyao** [comment]
all check passed, :-)
PTAL  @zimulala

**Pull request merged**